lib/data_update_scripts/20200218195023_index_chat_channel_memberships_to_elasticsearch.rb

```diff
@@ -1,7 +1,7 @@
 module DataUpdateScripts
   class IndexChatChannelMembershipsToElasticsearch
     def run
-      ChatChannelMembership.find_each(&:index_to_elasticsearch_inline)
+      # ChatChannelMembership.find_each(&:index_to_elasticsearch_inline)
```

Comment:
Is it feasible to just delete these files?

Reply:
@jgaskins (also answering [your question](https://github.com/forem/forem/pull/13606#discussion_r624192744) here @djuber)...the short of it is no.

Data Update Scripts are backed by the DB with the file names. If we delete the files, deploy it across all Forems, we'd have to make sure all the Forems DBs are in sync. It's happened before where that didn't go as planned so we resorted to commenting them out. Technically, I think a quick Data Update Script to delete other Data Update Scripts might work here as well. We could also tweak how Data Update Scripts work to introduce the concept of "deprecated" scripts.

We chose not to do that here because while it would be nice, it feels slightly out of scope for what is already a large and somewhat complex change here. We were hoping to get away with focusing on _just_ the removal of Elasticsearch.

FWIW, we have a list of cleanup items after Elasticsearch is gone (so many that is split into 2 notes [1](https://github.com/orgs/forem/projects/29#card-59639576) and [2](https://github.com/orgs/forem/projects/29#card-60117098)). We've added the removal of these scripts to that list since it's not absolutely needed to remove Elasticsearch.

Reply:
Also @jgaskins

> There seem to be a few places that can be mopped up (grep -ri elastic $(find . -name '*.rb') returns some results outside of data update scripts), but I'm assuming y'all have plans for that post-deploy.

Great call, I ran it as well and took a look. Those will all be updated when we address the follow up refactors:
- https://github.com/orgs/forem/projects/29#card-59639576
- https://github.com/orgs/forem/projects/29#card-60117098

Reply:
We have removed several DUS in the past. There's no really compelling reason for them to stick around after they ran across the fleet. Rails migrations are also backed by the DB but can eventually be removed since setup uses the dumped schema anyway. But we can of course do that later.

Reply:
I'm not sure we can actually remove the file:

https://github.com/forem/forem/blob/4aebfb62f25f825fa6fc235a5e56535d4121c0ca/app/models/data_update_script.rb#L23